ts/Core/Tooltip.ts

```diff
@@ -664,7 +666,7 @@ class Tooltip {
         const preferFarSide = !this.followPointer &&
             pick(
                 point.ttBelow,
-                !chart.inverted === flipped
+                polar ? (!polar && inverted) : !inverted === flipped
```

Comment:
This is quite a mind bender - what does it mean? What fails if we leave it like it was?

Reply:
Yes, I agree. When I'm looking at it now, it doesn't even make the logical sense (but it probably did before to me 😄). It's for the **inverted**, **polar**, and **yAxis.reversed** configuration (reversed y-axis basically). I changed it to `polar ? false : !inverted === flipped`. Thanks!